File: 92 - Orientacao a objetos/ruby/exec.rb
require File.expand_path('../joao', __FILE__)

# é o mostre que ensinei nos algoritimos
puts("Digite o CPF")
cpf = gets #leia que mostrei nos algoritimos;
puts("\n\n\n")

j = Joao.new(cpf)

j.nome = "sss"
puts j.nome

puts("CPF digitado" + j.cpf)
j.eu_sou()

puts("==========================");

j.metodo_sobresquever();

puts("==========================");

Joao.o_que_eu_sou();

File: 1l - Orientacao a objetos/java/comum/Exec.java
class Exec{
	public static void main(String[]args){
		// é o mostre que ensinei nos algoritimos
		System.out.println("Digite o CPF");
		String cpf = System.console().readLine(); // leia que mostrei nos algoritimos;
		System.out.printf("\n\n\n");

		Joao j = new Joao(cpf);
		System.out.println("CPF digitado" + j.cpf);
		j.euSou();

		// utilizando metodos agregados
		j.euSou("danilo agregado");


		System.out.println("==========================");

		j.metodoSobresquever();
		
		System.out.println("==========================");

		Joao.oQueEuSou();
		// j.nome = "João";
		// j.sobrenome = "da silva";
		// //j.cpf = "448.2339.566-08";

		// System.out.println(j.nomeCompleto());
		// j.completo();

		// //exec static
		// System.out.println(Joao.oQueEuSou());
	}
}